### PR TITLE
Add an alternative html parser

### PR DESCRIPTION
**Pull request by ayrat555**

### CODE REVIEW

**Review by pxp9 — APPROVED** [review]
LGTM

File: src/deliver/render_message.rs [hunk 1]
```diff
@@ -196,17 +212,32 @@ fn truncate(s: &str, max_chars: usize) -> String {
...
+    let cleaned = ac.replace_all(
         &string_without_html,
         &[" ", "&amp;", "&lt;", "&gt;", " ", " ", " ", " ", " "],
-    )
+    );
+
+    match html_parser {
+        HtmlParser::Nano => cleaned,
+        HtmlParser::Ammonia => {
+            let regexp = regex::Regex::new(r"(\n(\s)*(\n)(\s)*\n)").unwrap();
```
Comment (pxp9) [inline src/deliver/render_message.rs:239]: What is the reason for this unwrap ? 

Can be managed with `?` Operator ?

**Review by pxp9 — APPROVED** [review]

File: src/deliver/render_message.rs (hunk 1 above)
Comment (ayrat555) [reply src/deliver/render_message.rs:239]: because it's a hard-coded regexp and we're sure it's a valid one

Comment (pxp9) [reply src/deliver/render_message.rs:239]: Nice then

**Pull request closed without merging**